src/pages/hotel/booking/components/FinalScreen.tsx

```diff
@@ -0,0 +1,78 @@
+import Footer from '../../../dashboard/components/footer/Footer'
+import Header from '../../../dashboard/components/header/Header'
+import infoSVG from '../../../../../src/assets/svg/info.svg'
+import { FaHotel } from 'react-icons/fa'
+import { useEffect } from 'react'
+import './FinalScreen.scss'
+
+const FinalScreen = () => {
+  useEffect(() => {
+    window.history.pushState(null, '', window.location.href)
+    window.addEventListener('popstate', () => {
+      window.history.go(1)
+    })
+  }, [])
+
+  return (
+    <>
+      <div className="home-outline">
+        <Header />
+      </div>
+      <div>
+        <div className="booking-confirmation-panel">
+          <div className="info-outline">
+            <div className="info-image-container">
+              <img src={infoSVG} alt="" className="info-image" />
+            </div>
+            <div className="info-content">
+              Your trip has been successfully booked . Thank you for your
+              reservation.
+            </div>
+          </div>
+        </div>
+        <div className="booking-content">
+          <div className="booking-details-container">
+            <div className="booking-details">
+              <div id="content">
+                <FaHotel className="hotel-room-image" />
+              </div>
+              <div className="hotel-name">
+                <h1> Holiday Inn Express London Victoria</h1>
+                <div className="flex font-normal">
+                  <div className="font-medium text-base mt-0.5">Booked</div>
+                  <div className="booking-id">Booking ID - NHDESDDSDSDS73</div>
+                </div>
+              </div>
+              <div className="view-booking">
+                <button className="btn-viewbooking">VIEW BOOKING</button>
+              </div>
+            </div>
+          </div>
+          <div className="hotel-details-container">
+            <div className="mt-10 ml-28 space-x-40 flex text-gray-500 text-xl">
+              <h1>CHECK-IN</h1>
+              <h2>CHECK-OUT</h2>
+            </div>
+            <div className="ml-28 flex font-bold text-xl">
+              <div>29Mar&apos;22 Wed</div>
+              <div className="ml-28">31Mar&apos;22 Fri</div>
+              <div className="ml-28">
+                <FaHotel className="icon" />
+              </div>
+              <div className="ml-2 text-sky-500 font-medium text-lg">
+                1 Room(s),1 Night(s)
```

Comment:
1 Nights that you have to take from redux state. We already have days stores in redux state. Please check HotelDescription.tsx , how days taken and passed. Similiarly take it and instead hardcoding pass that value here